resolvers.ts

```diff
@@ -193,27 +194,143 @@ export const resolvers: Resolvers = {
-          args,
-          context
-        );
-        return formatSamples(samples);
+    samples: async (root, args, context) => {
+      const input = args.input;
+
+      // The comments in the formatUrlParams() call correspond to the line in the current
+      // codebase's callstack where the params are set, so help ensure we're not missing anything.
+      const { workflow_runs } = await get(
+        "/workflow_runs.json" +
+          formatUrlParams({
+            // index.ts
+            // const getWorkflowRuns = ({
+            mode: "with_sample_info",
+            //  - DiscoveryDataLayer.ts
+            //    await this._collection.fetchDataCallback({
+            domain: input?.todoRemove?.domain,
+            //  -- DiscoveryView.tsx
+            //     ...this.getConditions(workflow)
+            projectId: input?.todoRemove?.projectId,
+            search: input?.where?.name._like,
+            orderBy: input?.orderBy?.key,
+            orderDir: input?.orderBy?.dir,
+            //  --- DiscoveryView.tsx
+            //      filters: {
+            host: input?.where?.hostTaxon?.upstreamDatabaseIdentifier?._in,
+            locationV2: input?.where?.collectionLocation?._in,
+            taxon: input?.todoRemove?.taxons,
+            taxaLevels: input?.todoRemove?.taxaLevels,
+            time: input?.todoRemove?.time,
+            tissue: input?.where?.sampleType?._in,
+            visibility: input?.todoRemove?.visibility,
+            workflow: input?.todoRemove?.workflow,
+            //  - DiscoveryDataLayer.ts
+            //    await this._collection.fetchDataCallback({
+            limit: input?.todoRemove?.limit,
+            offset: input?.todoRemove?.offset,
+            listAllIds: input?.todoRemove?.listAllIds,
+          }),
+        args,
+        context
+      );
+      if (!workflow_runs?.length) {
+        return [];
       }
+
+      return workflow_runs.map((run): query_samples_items => {
+        const inputs = run.inputs;
+        const qualityMetrics = run.cached_results?.quality_metrics;
+        const sample = run.sample;
+        const sampleInfo = sample?.info;
+        const sampleMetadata = sample?.metadata;
+        return {
+          todoRemove: {
+            status: run.status,
+            startedAt: run.created_at,
+            creationSource: inputs?.creation_source,
+            workflowVersion: {
+              version: run.wdl_version,
+            },
+          },
+          id: sample?.id,
+          name: sampleInfo?.name,
+          notes: sampleInfo?.sample_notes,
+          collectionLocation: sampleMetadata?.collection_location_v2,
+          sampleType: sampleMetadata?.sample_type,
+          waterControl: sampleMetadata?.water_control,
+          hostTaxon: {
+            name: sampleInfo?.host_genome_name,
+          },
+          collection: {
+            name: sample?.project_name,
+            public: Boolean(sampleInfo?.public),
```

Comment:
do we not need the collection id?

Reply:
For displaying the sample rows, it seems like we don't